CHANGELOG.md

```diff
@@ -61,4 +62,9 @@ Remove `COMPRESSOR` variable from `CompressorFactory` and use `DEFLATE_COMPRESSO
 
 ### Security
 
+<<<<<<< HEAD
```

Comment:
@reta looks like you missed these conflicts :)

Reply:
Indeed, thanks a lot!